src/current/molt/migrate-failback.md

```diff
@@ -41,160 +67,276 @@ GRANT SELECT, INSERT, UPDATE, FLASHBACK ON migration_schema.orders TO MIGRATION_
 ~~~
 </section>
 
-## Configure failback
+## Configure replication
 
-Configure the MOLT Fetch connection strings and filters for `failback` mode, ensuring that the CockroachDB changefeed is correctly targeting your original source.
-
-### Connection strings
+When you run `replicator`, you can configure the following options for replication:
 
-In `failback` mode, the `--source` and `--target` connection strings are reversed from other migration modes: 
+- [Connection strings](#connection-strings): Specify URL‑encoded source and target connections.
+- [TLS certificate and key](#tls-certificate-and-key): Configure secure TLS connections.
+- [Tuning parameters](#tuning-parameters): Optimize failback performance and resource usage.
+- [Replicator metrics](#replicator-metrics): Monitor failback replication performance.
 
-`--source` is the CockroachDB connection string. For example:
+### Connection strings
 
-~~~
---source 'postgres://crdb_user@localhost:26257/defaultdb?sslmode=verify-full'
-~~~
+For failback, MOLT Replicator uses `--targetConn` to specify the original source database and `--stagingConn` for the CockroachDB staging database.
```

Comment:
Hmm this might be confusing now since we don't have to explain it in terms of source or target for the data load portion. I think it may be clearer here to describe the target connection as the destination you want the data to go from from Cockroach sources.